README.md

```diff
@@ -322,6 +323,25 @@ typedef NS_ENUM(NSInteger, NYTAdRequestState) {
 };
 ```
 
+## Boxing Enumerated Types
+
+Althought `enum`s are integral values, they cannot be directly used as [boxed literals](http://clang.llvm.org/docs/ObjectiveCLiterals.html#boxed-enums). So when creating a dictionary using `enum`s, they should be written as boxed literals instead of their corresponding `NSNumber` expression. 
```

Comment:
I’d beg to differ. Integral makes sense in this context:

![screen shot 2015-04-22 at 2 59 22 pm](https://cloud.githubusercontent.com/assets/14093/7282774/36265614-e900-11e4-8b36-9043f52376b0.png)